src/gcinfo/gcinfoencoder.cpp

```diff
@@ -1217,7 +1217,7 @@ void GcInfoEncoder::Build()
         // (after, of course, adding the size of the call instruction to get the return PC).
         callSite += m_pCallSiteSizes[callSiteIndex] - 1;
 
-        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite)) == callSite);
+        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite + 1)) == (callSite + 1));
```

Comment:
Thanks you for comment. Then, we need to revising macros DENORMALIZE_CODE_OFFSET and NORMALIZE_CODE_OFFERT for ARM, as in your PR 4481. 

But, it still confuses me that the value of ``callSite`` was odd when I checked at runtime. It would be very helpful if there is any document that states the purpose of these macros.